builtin/providers/aws/provider.go

```diff
@@ -147,6 +147,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_alb":                      dataSourceAwsAlb(),
 			"aws_alb_listener":             dataSourceAwsAlbListener(),
 			"aws_ami":                      dataSourceAwsAmi(),
+			"aws_autoscaling_groups":       dataSourceAwsAutoscalingGroups(),
```

Comment:
Should we name this `aws_autoscaling_group_names` since it doesn't return all the properties?